server/consumer.go

```diff
@@ -717,8 +717,7 @@ func (mset *stream) addConsumerWithAssignment(config *ConsumerConfig, oname stri
 	}
 
 	mset.mu.RLock()
-	s, jsa, cfg, acc := mset.srv, mset.jsa, mset.cfg, mset.acc
-	retention := cfg.Retention
+	s, jsa, cfg, acc := mset.srv, mset.jsa, mset.getCfg(), mset.acc
```

Comment:
`meet.getCfg()` may not even be needed here, since meet.setCfg() is still called under mset.mu.Lock().

Reply:
I have reverted that part in the new commit.